MVCForum.Core/ExtensionMethods/StringExtensionMethods.cs

```diff
@@ -0,0 +1,142 @@
+﻿namespace MvcForum.Core.ExtensionMethods
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Drawing;
+    using System.Globalization;
+    using System.Net;
+    using System.Text;
+    using Utilities;
+
+    public static class StringExtensionMethods
+    {
+        /// <summary>
+        ///     Returns an Image from an exteneral url
+        /// </summary>
+        /// <param name="url"></param>
+        /// <returns></returns>
+        public static Image GetImageFromExternalUrl(this string url)
+        {
+            var httpWebRequest = (HttpWebRequest)WebRequest.Create(url);
+
+            using (var httpWebReponse = (HttpWebResponse)httpWebRequest.GetResponse())
+            {
+                using (var stream = httpWebReponse.GetResponseStream())
+                {
+                    if (stream != null)
+                    {
+                        return Image.FromStream(stream);
+                    }
+                }
+            }
+            return null;
+        }
+
+        /// <summary>
+        ///     Turns a string to an array
+        /// </summary>
+        /// <param name="value"></param>
+        /// <param name="charactorToSplit"></param>
+        /// <param name="lowerCase"></param>
+        /// <returns></returns>
+        public static string[] ToArray(this string value, char charactorToSplit = ',', bool lowerCase = false)
```

Comment:
seems like we're always lower casing regardless of input, and `charactorToSplit` is going unobserverd also